### PR TITLE
Prevent null from being sent in team query

### PR DESCRIPTION
**Pull request by chiaberry**
Closes https://github.com/cityofaustin/atd-data-tech/issues/6943

Thank you Sergio for helping walk me through it. 

deployed at https://moped-test.austinmobility.io/moped/projects/new

### CODE REVIEW

File: moped-editor/src/views/projects/projectView/ProjectTeamTable.js [hunk 1]
```diff
@@ -42,7 +42,7 @@ const ProjectTeamTable = ({
   const classes = useStyles();
 
   const { loading, error, data, refetch } = useQuery(TEAM_QUERY, {
-    variables: { projectId },
+    variables: { projectId: projectId ?? 0 },
```
Comment (sergiogcx) [inline moped-editor/src/views/projects/projectView/ProjectTeamTable.js:45]: 👍

**Review by sergiogcx — APPROVED** [review]
Works great in moped-test! Thanks for putting this branch together 🙏 🚢 🚀

File: moped-editor/src/views/projects/projectView/ProjectTeamTable.js (hunk 1 above)
Comment (mateoclarke) [reply moped-editor/src/views/projects/projectView/ProjectTeamTable.js:45]: maybe it doesn't hurt to add a comment above this line that a `null` value bombs the app and help explains the fallback value of 0.

**Review by mateoclarke — APPROVED** [review]
🚢

**Comment by johnclary** [comment]
hooray! can we get this merged?

**Comment by chiaberry** [comment]
@amenity am I good to merge?

**Comment by mateoclarke** [comment]
Let's go ahead and merge Chia.

**Pull request merged**